### PR TITLE
Fix #7886: Add 1.57 translations.

### LINKED ISSUE #7886: Add 1.57 translations

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #7886 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Test Plan:

Verify that the "Private tabs" section in settings is translated.

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 1]
```diff
@@ -313,11 +322,14 @@
 /* The accessibility label for the contract address of a token / asset. */
 "wallet.contractAddressAccessibilityLabel" = "合约地址";
 
+/* A button title that user has clicked to copy some data to the users clipboard */
+"wallet.copiedToPasteboard" = "已複製";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]:
```suggestion
"wallet.copiedToPasteboard" = "已复制";
```

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 2]
```diff
@@ -901,6 +928,9 @@
 /* The label that is shown above the URL for the network on a request to switch networks or add a new network from a dapps website. */
 "wallet.networkURLTitle" = "网络 URL";
 
+/* A placeholder string that will be used on password text fields to create a new wallet */
+"wallet.newPasswordPlaceholder" = "輸入新密碼";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]:
```suggestion
"wallet.newPasswordPlaceholder" = "输入新密码";
```

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 3]
```diff
@@ -994,18 +1024,36 @@
...
+
 /* The label read out when a user is using VoiceOver and highlights the ellipsis button on the portfolio page */
 "wallet.otherWalletActionsAccessibilityTitle" = "其他钱包操作";
 
 /* The error message displayed when a user enters a password that does not meet the requirements */
 "wallet.passwordDoesNotMeetRequirementsError" = "密码长度至少为 8 个字符。";
 
 /* A placeholder string that will be used on password text fields */
-"wallet.passwordPlaceholder" = "密码";
+"wallet.passwordPlaceholder" = "請輸入密碼";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]:
```suggestion
"wallet.passwordPlaceholder" = "请输入密码";
```

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 4]
```diff
@@ -1075,35 +1129,41 @@
 /* A button on an account screen which when pressed presents a new screen to  rename the account */
 "wallet.renameButtonTitle" = "重命名";
 
+/* A label will be displayed when the repeat password field input is matched with the password field. */
+"wallet.repeatedPasswordMatch" = "符合！";
+
 /* A placeholder string that will be used on repeat password text fields */
-"wallet.repeatedPasswordPlaceholder" = "验证密码";
+"wallet.repeatedPasswordPlaceholder" = "重新輸入密碼";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]:
```suggestion
"wallet.repeatedPasswordPlaceholder" = "重新输入密码";
```

**Review by nuo-xu — CHANGES_REQUESTED** [review]

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 5]
```diff
@@ -1013,7 +995,7 @@
 "login.loginListSearchBarPlaceHolderTitle" = "Filtrele";
 
 /* The message displayed in alert when a user needs to set a passcode */
-"login.syncSetPasscodeAlertDescription" = "Senkronizasyon zincirini ayarlamak veya ayarları görmek için, öncelikle cihazınızda bir parola belirlemeniz gerekiyor.";
+"login.syncSetPasscodeAlertDescription" = "Senkronizasyon zincirine bir cihaz eklemek veya şifre senkronizasyonunu açmak veya kapatmak için, önce cihazınıza bir parola belirlemeniz gerekmektedir.";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings]: "Senkronizasyon zincirine bir cihaz eklemek veya şifre senkronizasyonunu açmak veya kapatmak için, önce cihazınıza bir pin kodu belirlemeniz gerekmektedir."

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 6]
```diff
@@ -1888,6 +1900,9 @@
 /* Settings privacy section title */
 "Privacy" = "Gizlilik";
 
+/* The message displayed in alert when a user needs to set a passcode */
+"privacy.tab.tray.passcode.alert" = "Özel gezinme moduna geçmek için, önce cihazınıza bir parola belirlemeniz gerekmektedir.";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings]: "Özel gezinme moduna geçmek için, önce cihazınıza bir pin kodu belirlemeniz gerekmektedir.";

**Review by soner-yuksel — APPROVED** [review]

**Review by nuo-xu — APPROVED** [review]

**Pull request merged**